app-admin/flannel/files/flanneld-rkt.service

```diff
@@ -27,10 +27,15 @@ ExecStart=/usr/bin/rkt run --net=host \
    --volume runsystemd,kind=host,source=/run/systemd,readOnly=false \
    --volume runflannel,kind=host,source=/run/flannel,readOnly=false \
    --volume ssl,kind=host,source=${ETCD_SSL_DIR},readOnly=true \
+   --volume certs,kind=host,source=/usr/share/ca-certificates,readOnly=true \
+   --volume resolv,kind=host,source=/etc/resolv.conf,readOnly=true \
    --mount volume=runsystemd,target=/run/systemd \
    --mount volume=runflannel,target=/run/flannel \
    --mount volume=ssl,target=${ETCD_SSL_DIR} \
+   --mount volume=certs,target=/etc/ssl/certs \
+   --mount volume=resolv,target=/etc/resolv.conf \
    ${FLANNEL_IMG}:${FLANNEL_VER} \
+   --exec /opt/bin/flanneld \
```

Comment:
Oh yeah, you're right. That's odd that we wrapped that in a shell. Maybe @eyakubovich remembers why that was done.